app/models/app_installation.rb

```diff
@@ -66,6 +66,14 @@ def github_client
     Octobox.installation_client(self.github_id)
   end
 
+  def write_issues?
+    permission_issues ? permission_issues == 'write' : false
+  end
+
+  def read_issues?
+    permission_issues ? permission_issues == 'read' : false
```

Comment:
I think this will return false if `permission_issues` is `'write'`, would be good to get a test for this, I believe we can just check to see if there's any value in `permission_issues` instead,

Reply:
I think `permission_issues` is 'read' by default (at least it looked that way on the live DB for octobox's repo. Will add a test.

Reply:
just ran this in irb and it seemed okay: maybe I am getting the type collapsing wrong?

```
irb(main):002:0> permission_issues = 'write'
=> "write"
irb(main):004:0> permission_issues ? permission_issues == 'read' : false
=> false
irb(main):005:0> permission_issues ? permission_issues == 'write' : false
=> true
```

Reply:
`permission_issues ? permission_issues == 'read' : false` => false is the problem, if the `permission_issues` field is `'write'`, then both `read_issues?` and `write_issues?` should return true.

Currently `read_issues?` returns false if the `permission_issues` field is `'write'`.